lib/tasks/grda_warehouse.rake

```diff
@@ -342,7 +342,7 @@ namespace :grda_warehouse do
 
     # Purge old soft-deleted records. Enable on production when we have confidence job is correct
     begin
-      PurgeSoftDeletedRecordsJob.perform_now(dry_run: false) if DateTime.current.hour == 5 && !Rails.env.production?
+      PurgeSoftDeletedRecordsJob.perform_later(dry_run: false) if DateTime.current.hour == 5
```

Comment:
Should I set a priority here or is it OK to enqueue as the default priority?

Reply:
This should probably get a low priority 12 or higher.